FlowCrypt/Controllers/Inbox/InboxViewDecorator.swift

```diff
@@ -30,6 +30,11 @@ extension InboxCellNode.Input {
 
         self.init(
             emailText: NSAttributedString.text(from: email, style: style, color: textColor),
+            countText: {
+                guard element.messageCount > 0 else { return nil }
+                let count = element.messageCount > 99 ? "99+" : String(element.messageCount)
+                return NSAttributedString.text(from: "(\(count))", style: style, color: textColor)
+            }(),
```

Comment:
If it's exactly `1` then we wouldn't have to print that number. Only need to print if it's >1.